src/NestField.tsx

```diff
@@ -25,16 +26,25 @@ const Nest = ({
 }: NestFieldProps) => {
 
   return (
-    <div
+    <Card
       {...filterDOMProps(props)}
     >
-      {label && <label>{label}</label>}
-      {children
-        ? injectName(name, children)
-        : fields?.map(key => (
-            <AutoField key={key} isDisabled={props.disabled} name={joinName(name, key)} {...itemProps} />
-          ))}
-    </div>
+      <CardBody>
+        {label && <label>{label}</label>}
```

Comment:
Checking if ionic supports h4 etc. We can wrap them in label itself.

Reply:
I could look into it, but I'm not too sure how we would get the information about the depth of the object. i.e. we would probably have to traverse the whole object to get the level (we could try using `traverse-js` to do this). But I'm not sure about the cost-benefit for this.

Ionic is a bit weird too, so even if you put those tags in, the components default the typography to the Ionic component's default and so the `<h1>`, `<h2>` tags and so on don't have any effect.

Reply:
ah.. that is the challenge. I will log that as issue and try it myself.